### DISCUSSION TITLE
How to copy-paste a rendered view?

### DISCUSSION THREAD

**Discussion by lucascervera** [stdword/logseq13-full-house-plugin#19] [Q&A] [answered]
I have created a template that works perfectly. Basically it renders a text substituting a variable with the corresponding value. I want to copy the generated text to the clipboard, but when i try to select the text i want to copy, the erendered text turns into "{{renderer :template-view, InnBoK type prompt template}}". Is there a way i can copy the rendered text to the clipboard?

**Comment by stdword** [comment, marked as answer]
@lucascervera You can go in 3 ways:

https://user-images.githubusercontent.com/1984175/233675155-b903c293-362e-4b89-ae4a-cbef62b80045.mp4

**Reply by lucascervera** [reply]
Thanks a lot. That was really helpful. Thanks for this wonderful pluggin.
